src/gui/SettingsWindow.cpp

```diff
@@ -408,8 +410,9 @@ void SettingsWindow::IntegerSettingVec(const std::string& property1, const std::
     ImGui::TableSetColumnIndex(1);
 
     int values[2] = {
-        _userConfiguration->getInt(property1, defaultValue1),
-        _userConfiguration->getInt(property2, defaultValue2)};
+        _safeInt(property1, defaultValue1),
+        _safeInt(property2, defaultValue2)
+    };
```

Comment:
Instead of this "_safeInt" function, it'd be way easier to simply catch the exception and use the default values if something is wrong, like this:

```c++
    int values[2];
    try
    {
        values[0] = _userConfiguration->getInt(property1, defaultValue1);
        values[1] = _userConfiguration->getInt(property2, defaultValue2);
    }
    catch (Poco::SyntaxException&)
    {
        values[0] = defaultValue1;
        values[1] = defaultValue2;
    }
```

This will also make sure that both values are using the default if _any_ property fails, and not an undefined mix of both.
Initialization isn't strictly necessary because either code path sets both values if successful.